tests/integration-tests/tests/dcv/test_dcv.py

```diff
@@ -130,3 +138,72 @@ def _check_security_group(region, cluster, port, expected_cidr):
     ips = response["SecurityGroups"][0]["IpPermissions"]
     target = next(filter(lambda x: x.get("FromPort", -1) == port, ips), {})
     assert_that(target["IpRanges"][0]["CidrIp"]).is_equal_to(expected_cidr)
+
+
+def _get_server_keys(hostname):
```

Comment:
What about moving these functions into a common/utils module, so that we can re-use it more easily if needed?